mobile-app/app/screens/AppNavigator/screens/Loans/screens/LoansTnCText.tsx

```diff
@@ -0,0 +1,83 @@
+
+import { useLanguageContext } from '@shared-contexts/LanguageProvider'
+
+export function useTnC (): string {
+    const { language } = useLanguageContext()
+
+    const text = {
+        en: tncText,
+        de: tncTextDe
+    }
+
+    return text[language]
```

Comment:
If my language is not english/german:
![telegram-cloud-photo-size-4-5965426866967919209-y](https://user-images.githubusercontent.com/85513542/180064722-cc757300-360b-45ab-9e1a-131af209da39.jpg)

Reply:
extremely good catch!